test-application/src/main/java/com/google/cloud/runtimes/MonitoringTestController.java

```diff
@@ -0,0 +1,75 @@
+package com.google.cloud.runtimes;
+
+import com.google.api.Metric;
+import com.google.cloud.monitoring.v3.MetricServiceClient;
+import com.google.monitoring.v3.*;
+import com.google.protobuf.util.Timestamps;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.io.IOException;
+
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+@RestController
+public class MonitoringTestController {
+
+    public static class MonitoringTestRequest {
+        private String name;
+        private Long token;
+
+        public void setToken(Long token) {
+            this.token = token;
+        }
+
+        public void setName(String name) {
+            this.name = name;
+        }
+    }
+
+    @RequestMapping(path = "/monitoring", method = POST)
+    public String handleMonitoringRequest(@RequestBody MonitoringTestRequest monitoringTestRequest) throws IOException {
+        MetricServiceClient.create()
+                .createTimeSeries(
+                        createTimeSeriesRequest(
+                                checkAndGetProjectIdFromEnv(),
+                                monitoringTestRequest.name,
+                                monitoringTestRequest.token));
+        return "OK";
+    }
+
+    private CreateTimeSeriesRequest createTimeSeriesRequest(String projectId, String metricType, Long metricValue) {
+        ProjectName projectName = ProjectName.create(projectId);
+        Metric metric = Metric.newBuilder()
+                .setType(metricType)
+                .build();
+        TimeInterval interval = TimeInterval.newBuilder()
+                .setEndTime(Timestamps.fromMillis(System.currentTimeMillis()))
+                .build();
+        TypedValue pointValue = TypedValue.newBuilder()
+                .setInt64Value(metricValue)
+                .build();
+        Point point = Point.newBuilder()
+                .setInterval(interval)
+                .setValue(pointValue)
+                .build();
+        TimeSeries ts = TimeSeries.newBuilder()
+                .setMetric(metric)
+                .addPoints(point)
+                .build();
+        return CreateTimeSeriesRequest.newBuilder()
+                .setNameWithProjectName(projectName)
+                .addTimeSeries(ts)
+                .build();
+    }
+
+    private String checkAndGetProjectIdFromEnv() {
+        String projectId = System.getenv("GOOGLE_CLOUD_PROJECT");
+        if (projectId == null) {
+            throw new IllegalArgumentException("Please set your project id! Usage: -DGOOGLE_CLOUD_PROJECT=YOUR_PROJECT_ID");
```

Comment:
well, yes, however I think this would be better to keep generic, e.g.: Please set your project id using the environment variable: GOOGLE_CLOUD_PROJECT
this change is coming up soon.